docs/helm-chart-submission.md

```diff
@@ -16,7 +21,39 @@ The following options are available for submitting a chart for Red Hat OpenShift
 
 > **_NOTE:_**  It is recommended when submitting a chart to submit chart source over a tarball. 
 
+> **_NOTE:_**  When submitting a Verification report only the report must be generated using the public url for the chart. 
+
 For more information on the submission process, see: [OpenShift Helm Charts Repository documentation](https://github.com/openshift-helm-charts/charts/blob/main/docs/README.md).
 
 For troubleshooting report related submission failures see: [Troubleshooting](./helm-chart-troubleshooting.md)
- 
\ No newline at end of file
+
+## Provider controlled delivery
+
+By default a submitted chart will be made available in the OpenShift Helm Chart Catalogue on successful certification. In some cases this is undesirable and can be prevented based on two conditions:
+
+1. When generating the Verification report the ```--provider-delivery``` flag is used, for example:
+    ```
+    $ podman run --rm -i                                  \
+          -e KUBECONFIG=/.kube/config                   \
+          -v "${HOME}/.kube":/.kube                     \
+          "quay.io/redhat-certification/chart-verifier" \
+          verify --provider-delivery                    \
+          <chart-uri>
+    ```
+    This ensures the [providerControlledDelivery annotation](helm-chart-annotations.md#providerControlledDelivery) is set to True in the verification report.
+
+1. The OWNERS file for the submitted chart in the [openshift helm charts github repository](https://github.com/openshift-helm-charts/charts) includes a ```providerDelivery``` attribute which is set to True, for example:
+```
+chart:
+  name: mychart
+  shortDescription: Test chart for testing chart submission workflows.
+publicPgpKey: null
+providerDelivery: True
+users:
+- githubUsername: myusername
+vendor:
+  label: redhat
+  name: Redhat
+```
+
+If these two conditions are met when the chart is submitted for certification, successful certification will not result in the chart being published in the OpenShift catalogue.
```

Comment:
If these preceding conditions are met when the chart is submitted for certification, successful certification will not result in the chart being published in the OpenShift catalog.

Reply:
Done.